### ISSUE TITLE
TypeNBT causes a warning due to type signature being incompatible with Scala 3

### ISSUE THREAD

**Issue by kory33** [kory33/s2mc-test#2]
```
[warn] An existential type that came from a Scala-2 classfile cannot be
[warn] mapped accurately to to a Scala-3 equivalent.
[warn] original type    : net.katsstuff.typenbt.NBTType.CovarObj[_$2] forSome _$2
[warn] reduces to       : net.katsstuff.typenbt.NBTType[?, ? <: net.katsstuff.typenbt.NBTTag.Aux[_$2]]
[warn] type used instead: net.katsstuff.typenbt.NBTType[?, ? <: net.katsstuff.typenbt.NBTTag.Aux[Any]]
[warn] This choice can cause follow-on type errors or hide type errors.
[warn] Proceed at own risk.
[warn] one warning found
```

**Comment by kory33** [comment]
Looks like `@nowarn` annotation does not suppress this. Probably we need to open a ticket on the upstream NBT library.

**Comment by kory33** [comment]
OK, TypeNBT published a scala 3 compatible version
https://github.com/Katrix/TypeNBT/commit/641dde8bf0878910ad456df7fd9274c995aa29c4

We should update the dependency and it should just work

**Comment by kory33** [comment]
Fixed by #67.
